semgrep/setup.py

```diff
@@ -106,6 +106,7 @@ def find_executable(env_name, exec_name):
         "ruamel.yaml==0.16.10",
         "tqdm>=4.46.1",
         "packaging>=20.4",
+        "pathspec>=0.8.0",
```

Comment:
What is the right steps to add a new third-party library?
The README said: "Runtime dependencies may not be installed via pipenv. They must be specified in setup.py."
So, I added this line.
However, I still failed the mypy import test.

Reply:
cc. @brendongo @underyx

Reply:
This looks correct to me. Dependencies for mainline functionality go in `setup.py`, while dev dependencies go in the `Pipfile`. For mypy you probably have to add the following to `mypy.ini`:

```
# pathspec
[mypy-pathspec.*]
ignore_missing_imports = True
```